frontend/app/src/people/widgetViews/OrganizationDetails.tsx

```diff
@@ -383,14 +385,14 @@ const OrganizationDetails = (props: { close: () => void; org: Organization | und
           <Button
             disabled={addWithdrawDisabled}
             text="Withdraw"
-            color="white"
+            color="withdraw"
             style={{ borderRadius: '5px' }}
             onClick={() => setIsOpenWithdrawBudget(true)}
           />
           <Button
             disabled={addBudgetDisabled}
             text="Deposit"
-            color="white"
+            color="success"
```

Comment:
is this a color?

Reply:
@kevkevinpal regarding earlier comment, we may have different shades of green so I name pertaining to the action. Let me know your thoughts

Reply:
ahh gotcha